R/ff_sync.R

```diff
@@ -27,8 +27,12 @@
 #' }
 #'
 #' @export
+
+# source(system.file("R", "load_config.R", package = "ForestForesight"))
+config <- ff_load_config()
+
 ff_sync <- function(ff_folder, identifier, download_model = FALSE, download_data = TRUE, download_predictions = FALSE, download_groundtruth = TRUE,
-                    bucket = "forestforesight-public", region = "eu-west-1", verbose = TRUE, sync_verbose = FALSE) {
+                    bucket = config$AWS_BUCKET, region = config$AWS_BUCKET_REGION, verbose = TRUE, sync_verbose = FALSE) {
```

Comment:
is AWS_BUCKET the name of the bucket? can you say AWS_BUCKET_NAME?

Reply:
yes